cli-rs/src/main.rs

```diff
@@ -41,8 +41,8 @@ struct Cli {
 
 #[derive(Subcommand)]
 enum Commands {
-    /// Compiles (no-std) rust code to riscv assembly.
-    /// Needs `rustup target add riscv32imac-unknown-none-elf`.
+    /// Compiles rust code to riscv assembly.
```

Comment:
actually this option ultimately compiles it to powdr-asm, no?

Reply:
Yes. Fixed.